shopfloor_mobile/static/wms/src/settings/workstation.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2021 Camptocamp SA (http://www.camptocamp.com)
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+export var Workstation = Vue.component("workstation", {
+    data: function() {
+        return {
+            usage: "workstation",
+            workstation_scanned: false,
+            scan_data: {},
+            scan_message: {},
+            search_input_placeholder: "Scan workstation code",
+        };
+    },
+    template: `
+        <Screen :screen_info="{title: $t('screen.settings.workstation.title'), klass: 'settings settings-workstation'}">
+            <template v-slot:header>
+                <user-information
+                    v-if="workstation_scanned"
+                    :message="scan_message"
+                    />
+            </template>
+            <searchbar
+                v-on:found="on_scan"
+                :input_placeholder="search_input_placeholder"
+                />
+
+            <div class="button-list button-vertical-list full">
+                <v-row align="center">
+                    <v-col class="text-center" cols="12">
+                        <btn-back/>
+                    </v-col>
+                </v-row>
+            </div>
+        </Screen>
+    `,
+    mounted() {
+        const odoo_params = {usage: this.usage};
+        this.odoo = this.$root.getOdoo(odoo_params);
+        if (this.$route.params.identifier) {
+            this.getData(this.$route.params.identifier);
+        }
+    },
+    methods: {
+        on_scan: function(scanned) {
+            this.odoo.call("setdefault", {barcode: scanned.text}).then(result => {
+                this.workstation_scanned = true;
+                if (result.error) {
```

Comment:
what are you missing here that justifies the change above?

Also, I don't think this piece of code is required.
I would a TODO for something better: when app config is loaded return a mapping of features that are enabled.
This way you could simply check `this.$root.appconfig.features.xyz`